webportal/package.json

```diff
@@ -68,6 +70,7 @@
     "build": "npm run clean && npm run preprocess && npm run webpack",
     "clean:dist": "npm run rimraf -- dist",
     "clean": "npm cache clean && npm run clean:dist",
+    "lint": "eslint src/**/*.js",
```

Comment:
Add `npm lint` in `"build:dev"`?

Reply:
Another question. eslint only checks the .js files. How about those '.ejs' files that also contain javascript code?